README.md

```diff
@@ -139,7 +161,7 @@ See [action/codecov-action](https://github.com/codecov/codecov-action) documenta
     - uses: codecov/codecov-action@v1.0.6
       with:
         token: ${{ secrets.CODECOV_TOKEN }}
-        file: ros_ws/lcov/total_coverage.info
+        file: 'ros_ws/lcov/total_coverage.info ros_ws/coveragepy/.coverage'
```

Comment:
Looking at the `codecov-action` code (here: https://github.com/codecov/codecov-action/blob/f532c3a1452359a7f96c37efc741537b80555c74/index.js#L79), I think this should work, but I have yet to confirm.

Reply:
So it doesn't seem to be working even if I use something like `'ros_ws/{lcov/total_coverage.info,coveragepy/.coverage}'`. I guess one could always use the codecov action twice. Therefore I'll just remove this change.